packages/cli/src/databases/postgresdb/migrations/1587669153312-InitialMigration.ts

```diff
@@ -14,6 +14,8 @@ export class InitialMigration1587669153312 implements MigrationInterface {
 			tablePrefix = schema + '.' + tablePrefix;
 		}
 
+		await queryRunner.query(`SET search_path TO ${schema};`);
```

Comment:
don't we have to do this in every migration?

Reply:
That's the beauty of it :)

Reply:
but it just gets executed for fresh DBs, am I mistaken?

Reply:
Good point! Please see latest commits. Re-ran `npm run test:postgres -- integration` and `npm run test:postgres:alt-schema -- integration` and both passing.